protocol/avdl/chat1/local.avdl

```diff
@@ -111,13 +111,27 @@ protocol local {
     union { null, Pagination } pagination;
   }
 
+  // This causes fetching to return N items, where N = IdeallyGetUnreadPlus +
+  // Unread, if AtLeast <= N <= AtMost, or one of the bounds if there are too
+  // many / too few unread items. i.e. (derived from chris's comment)
+  // collar(AtLeast, (IdeallyGetUnreadPlus + Unread), AtMost)
+  //
+  // By definition, one could use a same non-zero number for both AtLeast and
+  // AtMost to precisely control the number of items returned.
+  record NumLimit {
+    int IdeallyGetUnreadPlus;
```

Comment:
Is there a better name we could use here? I have no idea what this means by just reading the name.